packages/components/psammead-headings/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
```

Comment:
Just remembered that the Readme needs updating too, to remove reference to the `id` attribute being generated. https://github.com/bbc/psammead/blob/cabd0c0aff674831f916e1bc1da5988efe03739a/packages/components/psammead-headings/README.md#when-to-use-this-component